packages/extension-ui/src/components/ScrollWrapper.tsx

```diff
@@ -3,6 +3,16 @@ import styled from 'styled-components';
 
 import { ThemeProps } from '../types';
 
+const getScrollIntoViewCenter = (elem: HTMLElement) => {
```

Comment:
Just 
```suggestion
const scrollIntoViewCenter = (elem: HTMLElement) => {
```
I suppose? We're not getting anything really.